mods/d2k/weapons/missiles.yaml

```
@@ -161,8 +165,11 @@ DeviatorMissile:
 		-ImpactSounds:
 	Warhead@5OwnerChange: ChangeOwner
 		Range: 512
-		Duration: 375
+		Duration: 405
 		InvalidTargets: Infantry, Structure
 		ValidRelationships: Enemy, Neutral
+	Warhead@Trigger: GrantExternalCondition
+		Condition: unitCapturated
```

```suggestion
		Condition: unit-captured
```

spelling error, and please use the standard formatting